lib/string_basics.dart

```diff
@@ -2,6 +2,7 @@
 // All rights reserved. Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+import 'package:characters/characters.dart';
```

Comment:
I believe there's supposed to be a newline between package and relative imports (from https://dart.dev/guides/language/effective-dart/style#do-place-package-imports-before-relative-imports)

Reply:
Added new line.